lib/preneeds/models/attribute_types/xml_date.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module Preneeds
+  # This is a custom type class for ensuring Time is always coerced as UTC first then RFC1123
+  class XmlDate < Virtus::Attribute
+    def coerce(value)
+      return nil if value.to_s.empty?
+
+      begin
+        return Time.iso8601(value&.to_s).utc.strftime('%Y-%m-%d')
+      rescue ArgumentError
+        return value
```

Comment:
Not crazy about this rescue block, but I think it will probably be OK. Value would most likely be nil, I would think. Maybe that's OK. But doesn't hurt to show the error for debugging purposes maybe.

NITPICK.